.circleci/config.yml

```diff
@@ -95,7 +95,7 @@ jobs:
         command: cd ingestion-beam && mvn clean test
     - run:
         name: Report code coverage
-        command: bash <(curl -s https://codecov.io/bash) -F ingestion-beam
+        command: bash <(curl -s https://codecov.io/bash) -F beam
```

Comment:
why?

Reply:
Apparently, dashes are not allowed:

https://docs.codecov.io/docs/flags#section-how-to-create-flags

Reply:
codecov is currently ignoring this upload due to the bad flag name